analyzer/command/apk/apk.go

```diff
@@ -257,5 +272,9 @@ func (a alpineCmdAnalyzer) guessVersion(apkIndexArchive *apkIndex, originalPkgs
 }
 
 func (a alpineCmdAnalyzer) Required(targetOS types.OS) bool {
+	if builtinos.Getenv(skipAPKIndex) != "" {
```

Comment:
Can we disable this analyzer by default and enable it when a CLI option is passed? It is best if we can do it with the following option.
https://github.com/aquasecurity/trivy/blob/96af6dc49943731cfa398d1e9a4fa4d566566e84/internal/app.go#L126-L130